app/Transformers/UserHighlightsTransformer.php

```diff
@@ -67,13 +67,13 @@ public function transform(Highlight $highlight)
     private function checkColorPreference($highlight)
     {
         $color_preference = checkParam('prefer_color') ?? 'rgba';
-        if ($color_preference === 'hex') {
+        if ($color_preference === 'hex' && $highlight->color) {
```

Comment:
When I was testing I saw that some case the color object is not set and It was returning error.